js/src/elements/form-behavior.ts

```diff
@@ -395,6 +395,7 @@ export class FormBehaviorElement extends HTMLElement {
 	): Promise<FormValidationResult> {
 		const errors: FormValidationError[] = [];
 		const { errorAlert, errorList } = this.getErrorAlert(form);
+		let prevInputName: string = ''; // Used for Radios that share the same name
```

Comment:
If this is only used for radios, it should be named something that indicates that.

Reply:
bump

Reply:
This is outdated.  Is there more you want to change here?